### PR TITLE
be6 Create base code for Movie hive back end #6

### PR DESCRIPTION
**Pull request by ganesankarthikeyan**
Created skeleton for our watchhivebe.
Most of them are autogenerated.
Main things to notice
    Main class
    Pom.xml
    Readme
    Basic look at the structure.

https://github.com/dasamantaraoaditya/watchhivefe/issues/6#issue-915690660

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,19 @@ Watch-Hive backend-code goes in here.
 * Docs: https://github.com/dasamantaraoaditya/watchhivefe/wiki
 * TaskTracking:https://github.com/users/dasamantaraoaditya/projects/2 
 * Slack Channel : https://watch-hive.slack.com/
+
+## Configurations preferred for watchhivebe
+1) Make sure you have JDK 8 or 11. (JDK 8 is preffered)
```
Comment (daredevilteja) [inline README.md:13]: change the spelling of preferred

Comment (daredevilteja) [inline README.md:13]: Change the spelling of preferred

**Review by daredevilteja — CHANGES_REQUESTED** [review]
When I am starting the project then I am getting an error
Whitelabel Error page - is this an expected error

**Comment by dasamantaraoaditya** [comment]
> When I am starting the project then I am getting an error Whitelabel Error page - is this an expected error

Yes , i think that is expected as we dont have any api yet and it falls to default error page.
Except for the review comments above, everything else looks good to me.

File: README.md (hunk 1 above)
Comment (ganesankarthikeyan) [reply README.md]: resolved.

**Pull request merged**